procurement_service/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# (c) 2016 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "Procurement Service",
+    "version": "10.0.1.0.1",
```

Comment:
Why not `10.0.1.0.0` since this is the first version of the module for this repo.

Reply:
done